components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -71,6 +72,16 @@ func (hc *Client) ReleaseStatus(rname string) (string, error) {
 	return strings.Replace(statusStr, `\n`, "\n", -1), nil
 }
 
+//IsReleaseDeletable returns true for release that can be deleted
+func (hc *Client) IsReleaseDeletable(rname string) (bool, error) {
+	statusRes, err := hc.helm.ReleaseStatus(rname)
+	if err != nil {
+		return false, err
+	}
+
+	return statusRes.Info.Status.Code == release.Status_DEPLOYED || statusRes.Info.Status.Code == release.Status_FAILED, nil
```

Comment:
on install failure we can check if status is != DEPLOYED, so other statuses like SUPERSEDED,FAILED,PENDING(..) we can ignore,however, in the future upgrade they will be important